src/components/Profile/Profile.jsx

```diff
@@ -304,7 +306,7 @@ export default function Profile(props) {
               <div className="profile-row">
                 <div>
                   <p className="profile-sub-heading">Your Current Location:</p>
-                  {!isLocationLoading
+                  {!isLocationLoading && userLocation.length
```

Comment:
Are you checking here that the userLocation.length isn't null? If you're checking that it's not 0 can you make this userLocation.length != 0 so it's clearer?